x-pack/plugins/observability_solution/apm/server/utils/es_fields_mappings.ts

```diff
@@ -316,20 +315,17 @@ export const spanLinksDetailsMapping = (fields: Partial<Record<string, unknown[]
 
 // todo: pending #192337
 export const linkedParentsOfSpanMapping = (fields: Partial<Record<string, unknown[]>>) => {
-  return {
-    span: {
-      links: [
-        {
-          trace: {
-            id: normalizeValue<string>(fields[SPAN_LINKS_TRACE_ID]),
-          },
-          span: {
-            id: normalizeValue<string>(fields[SPAN_LINKS_SPAN_ID]),
-          },
-        },
-      ],
-    },
-  };
+  return (fields[SPAN_LINKS_TRACE_ID] as string[]).map((v, index) => {
```

Comment:
not sure if this works for OTel

Reply:
Thanks for scaffolding this. I don't see a way to test it against real data so far.